src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1543,7 +1543,20 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             if (encoding == AltNumpadEncoding::Unicode)
             {
                 // UTF-32 -> UTF-16
-                if (s.accumulator <= 0xffff)
+                if (s.accumulator == 0)
+                {
+                    // If the user pressed Alt + VK_ADD, then released Alt, they probably didn't intend to insert a numpad character at all.
+                    // Send any accumulated key events instead.

```

Comment:
We could potentially also get by, by passing through Alt/+ presses immediately without constraints and only switch to `s.active == true` once we encounter the first number/character. This requires us to add a 2nd `.active` member, because one of them will be required to track the state of the `_altNumpadState` (`.active` is already used in the if condition above and below).

Reply:
As it turns out, numpad+ is the same as +. 🤦